src/filesystem/interface.js

```diff
@@ -133,14 +136,14 @@ function FileSystem(options, callback) {
   options = options || {};
   callback = callback || defaultCallback;
 
-  var flags = options.flags || [];
-  var guid = options.guid ? options.guid : defaultGuidFn;
-  var provider = options.provider || new providers.Default(options.name || FILE_SYSTEM_NAME);
+  const flags = options.flags || [];
+  const guid = options.guid ? options.guid : defaultGuidFn;
+  const provider = options.provider || new providers.Default(options.name || FILE_SYSTEM_NAME);
   // If we're given a provider, match its name unless we get an explicit name
-  var name = options.name || provider.name;
-  var forceFormatting = flags.includes(FS_FORMAT);
+  const name = options.name || provider.name;
+  const forceFormatting = flags.includes(FS_FORMAT);
 
-  var fs = this;
+  let fs = this;
```

Comment:
This can likely be `const`.  The `fs` instance won't change during this scope, will it?